content/en/blog/_posts/2021-04-19-introducing-indexed-jobs.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Introducing Indexed Jobs"
+date: 2021-04-19
+slug: introducing-indexed-jobs
+---
+
+**Author:** Aldo Culquicondor (Google)
+
+Once you have containerized a non-parallel [Job](/docs/concepts/workloads/controllers/job/),
+it is quite easy to get it up and running on Kubernetes without modifications to
+the binary. In most cases, when running parallel distributed Jobs, you had
+to set a separate system to partition the work among the workers. For
+example, you could set up a task queue to [assign one work item to each
+Pod](/docs/tasks/job/coarse-parallel-processing-work-queue/) or [multiple items
+to each Pod until the queue is emptied](/docs/tasks/job/fine-parallel-processing-work-queue/).
+
+In the Kubernetes 1.21 release, SIG Apps introduces the Job _completion mode_,
+a configuration option that allows you to control how Pod completions affect the
```

Comment:
Actually, I'd write:
```suggestion
The Kubernetes 1.21 release introduces a new field to control Job _completion mode_,
```
The existing wording is fine too.

Reply:
Done